_posts/2020-07-14-newpipe-0.19.6-release.md

```diff
@@ -0,0 +1,74 @@
+
+## Bug fixes
+
+As mentioned early, the audio/video desync bug has been fixed. This bug was quite annoying to have to deal with when streaming video and now has been squashed. <sup>[@Stypox in #3837](https://github.com/TeamNewPipe/NewPipe/pull/3837)</sup>
+
+A crash that occurred when quickly swiping through search suggestions has been fixed. <sup>[@Stypox in #3759](https://github.com/TeamNewPipe/NewPipe/pull/3759)</sup>
+
+Channel names are now visible for all themes when in the background player. The text color used to be always black, which clearly did not work to well when using a dark theme. <sup>[@Stypox in #3822](https://github.com/TeamNewPipe/NewPipe/pull/3822)</sup>
+
+Queueing another video while having stuff paused in the popup player or background player no longer resumes that paused video. <sup>[@budde25 in #3787](https://github.com/TeamNewPipe/NewPipe/pull/3787)</sup>
+
+NewPipe can now handle deleted comments in PeerTube instead of crashing. <sup>[@wb9688 in #340](https://github.com/TeamNewPipe/NewPipeExtractor/pull/340)</sup>
+
+## Improvements
+
+When you get an error (🙁), there is now a simpler way to have your report formatted properly for a GitHub issue (no need to use the [Error report to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/)). Just press "Copy formatted report" in the app, then [make an issue](https://github.com/TeamNewPipe/NewPipe/issues/new?assignees=&labels=bug&template=bug_report.md) (after ensuring it hasn't already been reported!). <sup>[@TobiGr in #3579](https://github.com/TeamNewPipe/NewPipe/pull/3579)</sup>
+
+Speaking of errors, there used to be a functionality that would remove an item from your playing queue if there is an error with its playback. This led to a temporary internet instability to cause many items to fail and thus being removed from the queue, making it possible for many items to be falsely removed. This functionality has been removed to fix the problem. <sup>[@Stypox in #3704](https://github.com/TeamNewPipe/NewPipe/pull/3704)</sup>
+
+You're now able to long press on a video title to copy it to the clipboard. This is helpful when you are sharing a video somewhere where the link doesn't make an embed box to show other info about the video. <sup>[@adinilfeld in #3772](https://github.com/TeamNewPipe/NewPipe/pull/3772)</sup>
+
+There is now more proper structural support for YouTube Mixes (so Mixes will be available in a future update, *not this one*). <sup>[@wb9688 in #3441](https://github.com/TeamNewPipe/NewPipe/pull/3441)</sup>
+
+Also for YouTube, NewPipe supports `start=` timestamps in links now so if you are sent a specific time in a long stream it will start playing there. <sup>[@wb9688 in Extractor #366](https://github.com/TeamNewPipe/NewPipeExtractor/pull/366)</sup>
```

Comment:
Mention that `start=` is used by YT's embedded player

Reply:
```suggestion
Also for YouTube, NewPipe supports `start=` timestamps (used by YouTube's embedded player) in links now so if you are sent a specific time in a long stream it will start playing there. <sup>[@wb9688 in Extractor #366](https://github.com/TeamNewPipe/NewPipeExtractor/pull/366)</sup>
```